### ISSUE TITLE
Update usage documentation

### ISSUE THREAD

**Issue by richwatts0** [dividab/tsconfig-paths#187]
Your documentation for usage is really not clear enough.

`node -r tsconfig-paths/register {path/to/compiled/js.main.js}`

**Comment by jonaskello** [comment]
Do you mean in the README? I think that is exactly what is says there?

**Comment by richwatts0** [comment]
It needs to be more eli5 - another example it isn't immediately clear if this works with ES Modules?

**Comment by jonaskello** [comment]
PRs that improve the documentation is always welcome :-)
